src/drawing/DrawingList.tsx

```diff
@@ -31,22 +32,20 @@ export function sortDrawing({ target: targetA }: AnnotationDrawing, { target: ta
 
 export function DrawingList({ activeId = null, annotations, className, onSelect = noop }: Props): JSX.Element {
     const [isListening, setIsListening] = React.useState(true);
-    const rootElRef = React.createRef<SVGSVGElement>();
+    const [rootEl, setRootEl] = React.useState<DrawingSVGRef | null>(null);
 
     // Document-level event handlers for focus and pointer control
-    useOutsideEvent('mousedown', rootElRef, (): void => {
+    useOutsideEvent('mousedown', { current: rootEl }, (): void => {
```

Comment:
This object reference will not remain stable between renders. Is that okay? Should `useOutsideEvent` be updated to accept an element instead of a ref?

Reply:
I think it's ok. If the object is changed, `useEffect` will clean up the last effect and start an new effect with the new object. One concern is that it will do this in every render, which may cause performance issue.

Reply:
Updating `useOutsideEvent` will not resolve the performance issue mentioned above, right?

Reply:
I don't think re-running this per-render is something we should do. Updating `useOutsideEvent` to accept an element should fix the issue, since it should only re-run when the element reference changes.

Reply:
Got it, will update `useOutsideEvent`